data/crac-creation/crac-creator-csa-profiles/pom.xml

```diff
@@ -74,7 +74,6 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>farao-crac-impl</artifactId>
             <version>${project.version}</version>
-            <scope>runtime</scope>
```

Comment:
i think it's better to keep runtime here,  you need to provide the implementation at the runtime in the built but you don't need to make that code accessible.